pipelines/ocw/backup-ocw-content/README.md

```diff
@@ -0,0 +1,31 @@
+### Basic Idea
+
+Once a day we will copy the content from the real ocw s3 buckets to a backup bucket. In the event that something goes wrong with a release/mass publish we can update the fastly configs to point to the backup bucket and all will be magically well again.
+
+### Runbook for DR of ocw-content
+
+- Confirm the last execution of the relevant `backup-ocw-content` job in concourse.
+- IMPORTANT: Pause that pipeline so that no further executions will take place (which would sync the bad content from the real bucket over the backup bucket...). To  feel double safe, you can run a destroy-pipeline `fly` command on it as well.
+- Go to fastly.
+  - Navigate to the relevant service.
+  - Clone the active configuration.
+  - In your new clone, under `origins` and `hosts`, update references to `ocw-content-<draft|live>-<env>` to `ocw-content-backup-<draft|live>-<env>`.
```

Comment:
Add:
Make sure that you update the "Override host", "SNI Host", and "Certificate Host" sections so that they match the value in the origin host configuration.